CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,89 @@
+- Reduce Antrea cni binary size by removing unnecessary import packages. ([#6038](https://github.com/antrea-io/antrea/pull/6038), [@tnqn])
+- Strip all debug symbols from Go binaries by default. ([#6035](https://github.com/antrea-io/antrea/pull/6035), [@antoninbas])
+- Skip loading `openvswitch` kernel module if it's already built-in. ([#5979](https://github.com/antrea-io/antrea/pull/5979), [@antoninbas])
+- Persist TLS certificate and key of antrea-controller and periodically sync the CA cert to improve robustness. ([#5955](https://github.com/antrea-io/antrea/pull/5955), [@tnqn])
+- Add more validations for ExternalIPPool CRD to improve robustness. ([#5898](https://github.com/antrea-io/antrea/pull/5898), [@aroradaman])
+- Add L7 Network Policy logs for `allowed` HTTP traffic. ([#6014](https://github.com/antrea-io/antrea/pull/6014), [@qiyueyao])
+- Update maximum number of buckets to 700 in OVS group add/insert_bucket message. ([#5942](https://github.com/antrea-io/antrea/pull/5942), [@hongliangl])
+- Add a flag for antctl to print OVS table names when users run `antctl get ovsflows --table-names-only`. ([#5895](https://github.com/antrea-io/antrea/pull/5895) [#6100](https://github.com/antrea-io/antrea/pull/6100), [@luolanzone])
+- Upgrade K8s libraries to v0.29.2. ([#5843](https://github.com/antrea-io/antrea/pull/5843), [@hjiajing])
+
+### Fixed
+
+- Fix a race condition in antrea-agent Traceflow controller when a tag is associated again with a new Traceflow before the old Traceflow deletion event is processed. ([#5954](https://github.com/antrea-io/antrea/pull/5954), [@tnqn])
+- Change the maximum flags from 7 to 255 to fix the wrong TCP flags validation issue in Traceflow CRD. ([#6050](https://github.com/antrea-io/antrea/pull/6050), [@gran-vmv])
+- Use 65000 MTU upper bound for interfaces in encap mode in case of large packets being dropped unexpectedly. ([#5997](https://github.com/antrea-io/antrea/pull/5997), [@antoninbas])
+- Install multicast related iptables rules only on IPv4 chains to fix the antrea-agent initialization failure occurred when the Multicast feature is enabled in dual-stack clusters. ([#6123](https://github.com/antrea-io/antrea/pull/6123), [@wenyingd])
+- Remove incorrect AntreaProxy warning on Windows when `proxyAll` is disabled. ([#6242](https://github.com/antrea-io/antrea/pull/6242), [@antoninbas])
+- Explicitly set kubelet's log files in Prepare-Node.ps1 on Windows to include the log files in support bundle collections. ([#6221](https://github.com/antrea-io/antrea/pull/6221), [@wenyingd])
+- Do not try to update the type of Secret in `selfSignedCertProvider`, so Antrea can reuse the existing Opaque type of Secret with a user-provided certificate. ([#6205](https://github.com/antrea-io/antrea/pull/6205), [@tnqn])
```

Comment:
This is a follow-up PR of a change in this release, so no need to mention it as a fixed